### PR TITLE
Add Getting Started Guide to UG

### LINKED ISSUE #283: UG - Quick Start Guide

### PR DESCRIPTION
**Pull request by B-enguin**
Edit UG to include new guide for new users to get started with HomeBoss. Includes installation, application layout, command format, and short tutorial.

Edit CSS Styling to make code blocks more visible, begin migration to syntax highlighting for code blocks.

Update Hyperlink in Site Map for Getting Started.

Closes #283.

### CODE REVIEW

File: docs/images/getting-started/HomePage.png
Comment (juliusgambe) [inline docs/images/getting-started/HomePage.png:1]: I think it should be Customer/Delivery List *Panel*?

**Review by juliusgambe — COMMENTED** [review]
I only looked through the annotations.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
...
 
-To get started, you will need to create an account and download the software. Once you have done that, you can follow
-the instructions in the Getting Started Guide to start using the software.
+## Prerequisites
+Ensure that you have `Java 11`{.swift} or above installed on your computer. If you are unsure how to do so you may view this
+helpful [guide](https://www.java.com/en/download/help/version_manual.html).
+
+<box type="tip" background-color="#d9edf7" border-color="#bce8f1" icon=":bulb:">
+
+**Tip**: You may find the following links helpful should you need to install `Java 11`{.swift}
```
Comment (juliusgambe) [inline docs/UserGuide.md:114]: **Tip**: -> **Tip:**?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
...
+
+Take the [Update Details Command](#update-details) for example:
+
+```swift
+update [--user USERNAME] [--password PASSWORD --confirmPass CONFIRM_PASSWORD] [--secretQn SECRET_QUESTION --answer ANSWER]```
+```
+
+* The **Command Phrase** is `update`{.swift}
+* The **Parameters** are the words in `UPPER_CASE`{.swift}, for example `USERNAME`{.swift} or `PASSWORD`{.swift}
+* The **Prefixes** are special keywords that a preceded by `--`{.SWIFT}, such as `--user`{.swift} or
```
Comment (zhonghan721) [inline docs/UserGuide.md]: Is there a typo here (`that are preceded` and not `a`), and would `and` be more suitable as compared to `or`?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
...
+
+**Note**:
+* Command Phrase are keywords or short phrases that are unique to each Command.
+* Words that are in `UPPER_CASE`{.swift} are parameters that are to be supplied by you.
+* Parameters that are preceded by a prefix must be supplied after that prefix, e.g. `--name DELIVERY_NAME`{.swift}
+  must be given as `--name deliveryName`{.swift} and not `deliveryName`{.swift} or `--name`{.swift}
+* Items that are placed in square brackets are optional, e.g. `DELIVERY_ID [--name DELIVERY_NAME]`{.swift} can either be
+  `1`{.swift} or `1 --name deliveryName`{.swift}.
+* However, items that are grouped together in square brackets must all be provided together, 
+  e,g, for `[--password PASSWORD --confirmPass CONFIRM_PASSWORD]`{.swift} both `PASSWORD`{.swift} and
```
Comment (zhonghan721) [inline docs/UserGuide.md]: Would it be `e.g.` instead?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -34,8 +34,209 @@ your operations. Within these pages, you'll find detailed coverage of the follow
...
+![CustomerAdded](images/getting-started/CustomerAdded.png){style="display: block; margin: 0 auto; box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);border-radius: 5px;"}
+
+6. Now, let's add a new delivery for Gabriel into HomeBoss, suppose Gabriel ordered some furniture.
+
+> `DELIVERY_NAME`: furniture
+> `DELIVERY_DATE`: 2023-12-03
+
+8. Type `delivery add furniture --customer CUSTOMER_ID --date 2023-12-03`{.swift}, replacing `CUSTOMER_ID`{.swift} with 
+   Gabriel's ID, and hit enter.
+9. new Delivery should have been added for Gabriel, you should see a new Delivery similar to the image below.
```
Comment (zhonghan721) [inline docs/UserGuide.md]: Should it start as a sentence?

**Review by zhonghan721 — COMMENTED** [review]
Some general comments

File: docs/UserGuide.md (hunk 2 above)
Comment (B-enguin) [reply docs/UserGuide.md]: Good catch

File: docs/UserGuide.md (hunk 4 above)
Comment (B-enguin) [reply docs/UserGuide.md]: Good catch

File: docs/UserGuide.md (hunk 3 above)
Comment (B-enguin) [reply docs/UserGuide.md]: Good catch

**Review by juliusgambe — APPROVED** [review]

**Pull request merged**